docs_app/content/tutorial/6-chaining-operators.md

```diff
@@ -0,0 +1,145 @@
+}
+
+// or a more readable version (this is identical in behavior):
+
+function pipe(...fns) {
+    return (input) => {
+        let result = input;
+        for (const fn of fns) {
+            result = fn(input);
+        }
+        return result;
+    }
+}
+```
+
+There are edge cases to cover, of course, like zero arguments, but overall this is the concept.
+
+## Piping Operators
+
+Since operators are all higher order functions that return unary functions of the shape `(source: Observable<A>) => Observable<B>`, that means they can be combined with [`pipe`](API).
+
+```ts
+import { pipe } from 'rxjs';
+import { map, filter } from 'rxjs/operators';
+
+// Here we're combining 3 operators into a single operator function
+// that is `(source: Observable<Apple>) => Observable<PeeledAppleSlice[]>`
+const processApples = pipe(
+    // `(source: Observable<Apple>) => Observable<Apple>`
+    filter(apple => apple.quality === 'GOOD'),
+    // `(source: Observable<Apple>) => Observable<PeeledApple>`
+    map(apple => createPeeledApple(apple)),
+    // `(source: Observable<PeeledApple>) => Observable<PeeledAppleSlice[]>`
+    map(peeledApple => slice(peeledApple))
+);
+```
+
+Note that `processApples` above is **NOT** an observable. It's just a function. It's a function built from three other functions that were returned by calls to [`filter`](API), [`map`](API), and another [`map`](API), in that order. You can now use it like so:
+
+```ts
+import { pipe } from 'rxjs';
+import { map, filter } from 'rxjs/operators';
+
+const apples$: Observable<Apple> = getApples();
+
+const processApples = pipe(
+    filter(apple => apple.quality === 'GOOD'),
+    map(apple => createPeeledApple(apple)),
+    map(peeledApple => slice(peeledApple))
+);
+
+processApples(apples$).subscribe(console.log);
+```
+
+This is nice, because you can reuse the operator function `processApples`, however, it's somewhat cumbersome to read, and unergonomic, so for convenience, RxJS has added a [`pipe` method](API) directly to [`Observable`](API). With the observable pipe method, the above is equivalent to:
+
+```ts
+import { map, filter } from 'rxjs/operators';
+
+const apples$: Observable<Apple> = getApples();
+
+apples$.pipe(
+    filter(apple => apple.quality === 'GOOD'),
+    map(apple => createPeeledApple(apple)),
+    map(peeledApple => slice(peeledApple))
+)
+.subscribe(console.log);
+```
+
+### The Pipeline Operator ECMAScript Proposal
+
+A notable development in the wild is the [ECMAScript Pipeline Operator Proposal](LINK) that is currently being examined by the [TC39](LINK), which is the basically the JavaScript standards committee. All forms of this operator (JavaScript operator, sorry I know that is confusing), do effectively the same thing as what the simple `pipe` function does above, however, it's done in a way that is more readable, universal, and easier to flow types through, than what we're currently doing in RxJS.
```

Comment:
Instead of "this", which can be confusing, consider specifying the specific thing.

this operator => the JavaScript pipe operator